src/jit/importer.cpp

```diff
@@ -5757,8 +5776,35 @@ GenTreePtr Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolve
                                     new (this, GT_CNS_INT) GenTreeIntCon(TYP_INT, pFieldInfo->offset, fs));
             }
             break;
+        }
+        case CORINFO_FIELD_STATIC_READYTORUN_HELPER:
+        {
+#ifdef FEATURE_READYTORUN_COMPILER
+            noway_assert(opts.IsReadyToRun());
+            CORINFO_GENERICHANDLE_RESULT embedInfo;
+            info.compCompHnd->embedGenericHandle(pResolvedToken, FALSE, &embedInfo);
+            assert(embedInfo.lookup.lookupKind.needsRuntimeLookup);
+
+            GenTreePtr      ctxTree = getRuntimeContextTree(&embedInfo.lookup);
+            GenTreeArgList* args    = gtNewArgList(ctxTree);
 
+            unsigned callFlags = 0;
+
+            if (info.compCompHnd->getClassAttribs(pResolvedToken->hClass) & CORINFO_FLG_BEFOREFIELDINIT)
+            {
+                callFlags |= GTF_CALL_HOISTABLE;
+            }
+
+            op1 = gtNewHelperCallNode(CORINFO_HELP_READYTORUN_STATIC_BASE, TYP_BYREF, callFlags, args);
+
+            op1->gtCall.setEntryPoint(pFieldInfo->fieldLookup);
```

Comment:
The latest commit lost the part where we add the field offset to the returned base. The helper getting called is good, but we need a `gtNewOperNode(GT_ADD, ...)` that take the result of the call and adds the offset. (I guess that part will be just like above in the `CORINFO_FIELD_STATIC_SHARED_STATIC_HELPER` case).